networking/multiple_networks/understanding-user-defined-network.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="understanding-user-defined-networks"]
+= Understanding user-defined networks
+include::_attributes/common-attributes.adoc[]
+:context: understanding-user-defined-networks
+
+toc::[]
+
+:featurename: {`UserDefinedNetwork`}
+include::snippets/technology-preview.adoc[]
+
+The OVN-Kubernetes CNI plugin has supported `layer3`, `layer2`, and `localnet` topology types for creating additional networks in which secondary interfaces of pods are attached, and for a primary network a pod only `layer3` topology type was supported.
```

Comment:
the first sentence seems off... can you please double check my initial comment and fix it,, the second half is not making sense

Reply:
updated